### PR TITLE
Use the new coarse timer module to ensure we don't violate the timing restrictions

### PR DESCRIPTION
**Pull request by nomis52**
This maintains 5.8 ms from the end of the DUB frame to the start of another frame.

### CODE REVIEW

File: firmware/src/transceiver.c [hunk 1]
```diff
@@ -265,7 +268,7 @@ void __ISR(_TIMER_1_VECTOR, ipl6) Transceiver_TimerEvent() {
     SYS_INT_SourceEnable(INT_SOURCE_USART_1_TRANSMIT);
     PLIB_TMR_Stop(TMR_ID_1);
   } else if (g_transceiver.state == TRANSCEIVER_RECEIVING) {
-    // BSP_LEDToggle(BSP_LED_2);
+    // BSP_LEDToggle(BSP_LED_1);
```
Comment (peternewman) [inline firmware/src/transceiver.c]: Remove please.

File: firmware/src/transceiver.c [hunk 2]
```diff
@@ -401,6 +407,7 @@ void __ISR(_UART_1_VECTOR, ipl6) Transceiver_UARTEvent() {
     }
     SYS_INT_SourceStatusClear(INT_SOURCE_USART_1_RECEIVE);
   } else if (SYS_INT_SourceStatusGet(INT_SOURCE_USART_1_ERROR)) {
+    // BSP_LEDToggle(BSP_LED_3);
```
Comment (peternewman) [inline firmware/src/transceiver.c]: Remove please.

File: firmware/src/transceiver.c [hunk 3]
```diff
@@ -415,10 +422,13 @@ void __ISR(_UART_1_VECTOR, ipl6) Transceiver_UARTEvent() {
...
         } else {
           // In break, stop the timer, and start a max-break timer.
+          PLIB_TMR_Stop(TMR_ID_1);
+          PLIB_TMR_Counter16BitClear(TMR_ID_1);
           PLIB_TMR_PrescaleSelect(TMR_ID_1 , TMR_PRESCALE_VALUE_1);
           PLIB_TMR_Period16BitSet(TMR_ID_1,
                                   g_transceiver.rdm_response_max_break_ticks);
-          // BSP_LEDToggle(BSP_LED_3);
+          PLIB_TMR_Start(TMR_ID_1);
+          // BSP_LEDToggle(BSP_LED_1);
```
Comment (peternewman) [inline firmware/src/transceiver.c]: Remove please

File: firmware/src/transceiver.c (hunk 1 above)
Comment (nomis52) [inline firmware/src/transceiver.c]: Done in the next PR

**Comment by peternewman** [comment]
Retrospective LGTM

**Pull request merged**